pyproject.toml

```
@@ -56,6 +56,15 @@ tox = "^3.24"
 click = "^8.1"
 Mako = "^1.2"
 deprecation = "^2.1"
+ni-python-styleguide = "^0.4"
+
+[tool.black]
+line-length = 100
+exclude = ["generated/",
```

specifying `exclude` can make black take a long time due to it now scanning a local .venv.
```suggestion
extend_exclude = ["generated/",
```